letsencrypt/cli.py

```diff
@@ -738,8 +756,16 @@ def determine_help_topics(self, chosen_topic):
             return dict([(t, t == chosen_topic) for t in self.help_topics])
 
 
-def create_parser(plugins, args):
-    """Create parser."""
+def parse_args(plugins, args):
```

Comment:
I don't really like having two functions / methods in the same file with the same name.  Maybe call this one `prepare_and_parse_args`?